reana_client/cli/__init__.py

```diff
@@ -44,6 +44,7 @@ class ReanaCLI(click.Group):
         workflow.interactive_group,
         files.files_group,
         secrets.secrets_group,
+        retention_rules.retention_rules_group,
```

Comment:
Minor: currently retention rules group description is displayed in the very bottom while calling `reana-client --help`. We should think if it would logically be better placed next to some other group, e.g. under `Workspace file management commands` (since it's related with workspace cleanup)? @tiborsimko

Reply:
Yes, the new section could come right after "Workspace file management commands". Also we can call it "Workspace file retention commands" to indicate close relationship.  (Otherwise calling it just retention rules, without specifying to what these rules apply, may be ambiguous.)